packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -814,23 +814,11 @@ var ReactDOMFiberComponent = {
     lastRawProps: Object,
     nextRawProps: Object,
   ): void {
-    var wasCustomComponentTag = isCustomComponent(tag, lastRawProps);
-    var isCustomComponentTag = isCustomComponent(tag, nextRawProps);
-    // Apply the diff.
-    updateDOMProperties(
-      domElement,
-      updatePayload,
-      wasCustomComponentTag,
-      isCustomComponentTag,
-    );
-
     // TODO: Ensure that an update gets scheduled if any of the special props
     // changed.
     switch (tag) {
       case 'input':
-        // Update the wrapper around inputs *after* updating props. This has to
-        // happen after `updateDOMProperties`. Otherwise HTML5 input validations
-        // raise warnings and prevent the new value from being assigned.
+        // Update the wrapper around inputs *after* updating props.
```

Comment:
this comment isn't correct anymore since you've moved the `updateDOMProperties` call below it. We would need to confirm that the original issue with validations firing doesn't occur. Can you write a dom fixture that demonstrates the issue and that it doesn't occur after this change?

@nhunzaker Any suggestions on how to write that, or what conditions triggered it?